Loop/Extensions/Environment+Dismiss.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Environment+Dismiss.swift
+//  Loop
+//
+//  Created by Michael Pangburn on 4/15/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+
+private struct PresentationDismissalKey: EnvironmentKey {
+    static let defaultValue = {}
+}
+
+
+extension EnvironmentValues {
+    var dismiss: () -> Void {
```

Comment:
Interesting pattern here of providing callbacks via the environment. I could see this helping out when we need to signal up the stack a ways, like you're doing here.

Reply:
I find it kinda funky that SwiftUI offers a way to dismiss modals it presents itself (there's an environment binding called `presentationMode`), but no way to dismiss from the top-level in a modal presented via UIKit—this felt like a fairly lightweight solution.